share/spice/similar_sites/similar_sites.css

```diff
@@ -1,20 +1,16 @@
-.zci--similar_sites #hidden {
+.zci--similar_sites .hidden, .zci--similar_sites .chomp--link__ls {
```

Comment:
There's no ```.hidden``` in the current versions of <a href="https://duckduckgo.com/s956.css">s956.css</a> or <a href="https://duckduckgo.com/t956.css">t956.css</a>. Maybe in a to-be-released version?